plugins/view-transitions/includes/settings.php

```diff
@@ -246,27 +255,31 @@ static function (): void {
 	);
 
 	$fields = array(
-		'default_transition_animation' => array(
+		'default_transition_animation'          => array(
 			'title'       => __( 'Default Transition Animation', 'view-transitions' ),
 			'description' => __( 'Choose the animation that is used for the default view transition type.', 'view-transitions' ),
 		),
-		'header_selector'              => array(
+		'default_transition_animation_duration' => array(
```

Comment:
The new input is currently set up as a text field and allows users to enter string values. If someone enters a non-numeric string, it will return an integer value of 0. We should either update it to use a number input field or sanitize the input to ensure only numeric values are stored.